packages/fiori/src/types/BarDesign.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @lends sap.ui.webcomponents.fiori.types.BarDesign.prototype
+ * @public
```

Comment:
Please use the same annotations as in "main"

/**
 * Different Bar designs.
 *
 * @class
 * @enum {string}
 * @public
 * @author SAP SE
 * @alias sap.ui.webcomponents.fiori.types.BarDesign
 
 
 This applies to all types

Reply:
Will do it, my bad!